prolif/interactions/interactions.py

```diff
@@ -131,6 +133,183 @@ def __init__(
 )
 
 
+class ImplicitHBAcceptor(Distance):
```

Comment:
rather than copy-pasting this and the `_get_radii_sum` method from the vdwcontact class, it would be better to inherit directly from it in addition to `Distance`:
```suggestion
class ImplicitHBAcceptor(Distance, VdWContact):
```
and reuse the appropriate initialisation methods for the different parameters